forte/data/data_store.py

```diff
@@ -313,52 +399,82 @@ def delete_entry(self, tid: int):
             tid (int): Unique id of the entry.
         """
         # We retrieve the entry data from `__entry_dict` using tid. We get its
-        # `entry_type`, `type_id`, `begin` and `end` indices. we remove the
-        # entry from `__entry_dict` when its information is retrieved. We
-        # find the
-        # `entry_type` sortedlist using `type_id`. We bisect the sortedlist to
-        # find the index of entry data. Then, `type_id` and index are passed to
-        # _delete_entry_by_loc.
+        # `entry_type`, `type_id`. If it is an Annotation-Like entry, we obtain
+        # `begin` and `end` indices and use bisection to locate the `index_id`.
+        # If it is not, we obtain the 'index_id' directly from `__entry_dict`.
+        # We remove the entry from `__entry_dict` when its information is
+        # retrieved. Then, the `type_id` and the `index_id` are passed to
+        # `_delete_entry_by_loc`.
         raise NotImplementedError
 
     def _delete_entry_by_loc(self, type_id: int, index_id: int):
         r"""It removes an entry of `index_id` by taking both the `type_id`
         and `index_id`. Called by `delete_entry()`.
 
         Args:
-            type_id (int): The index of the sortedlist in `self.__elements`.
-            index_id (int): The index of the entry in the sortedlist.
+            type_id (int): The index of list in `self.__elements`.
+            index_id (int): The index of the entry in the list.
         """
-        # We then remove the entry data from `entry_type` sortedlist.
+        # We then remove the entry data from `entry_type` list.
         raise NotImplementedError
 
     def get_entry(self, tid: int) -> Tuple[List, int, int]:
         r"""This function finds the entry with `tid`. It returns the entry,
-        its `type_id`, and the index in the `entry_type` sortedlist.
+        its `type_id`, and the index in the `entry_type` list.
 
         Args:
             tid (int): Unique id of the entry.
 
         Returns:
             The entry which `tid` corresponds to, its `type_id` and its index
-            in the `entry_type` sortedlist.
-
+            in the `entry_type` list.
         """
+        # If the entry is an annotation, bisect the annotation sortedlist
+        # to find the entry. May use LRU cache to optimize speed.
+        # Otherwise, use `index_id` to find the index of the entry.
         raise NotImplementedError
 
     def get(
         self, entry_type: Union[str, Type[EntryType]], **kwargs
```

Comment:
What is the kwargs for? it is not used

Reply:
We may want to support: get within a `begin`, `end` range, etc.

Reply:
Having `kwargs` will likely be a bad design since it means we don't have an idea of what we will support. In Forte we tend to avoid any kwargs

Reply:
Sure, I will remove it.